assets/prometheus-k8s/rules.yaml

```diff
@@ -800,17 +800,17 @@ spec:
     - alert: KubeClientCertificateExpiration
       annotations:
         message: A client certificate used to authenticate to the apiserver is expiring
-          in less than 7 days.
+          in less than 0 days.
```

Comment:
we should change the alert message format to use hours rather than days, as this is not quite helpful right now.

Reply:
Sure, I will prepare a patch upstream in the kubernetes-mixins.

Reply:
Xref: https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/168